routing/endpointregistry_test.go

```diff
@@ -114,6 +114,34 @@ func TestDoRemovesOldEntries(t *testing.T) {
 	assert.Equal(t, int64(0), mRemoved.InflightRequests())
 }
 
+func TestDoDoesNothingForUnrelatedBackends(t *testing.T) {
```

Comment:
Its better to test that it does X. Maybe also add network and loadbalancer route and then check that metrics were only added for them (see https://github.com/zalando/skipper/pull/2869/files#r1453615534)

Reply:
`TestEndpointRegistryPostProcessor`

Reply:
Okay, I created LB backend with 2 endpoints, network backend with single host and left routes with all other backend types.
After that, I check that there're only 3 entries in endpointregistry, and they correspond to 2+1 hosts created above.